### PR TITLE
REL-2833: Add Phoenix support to QPT

### PR DESCRIPTION
**Pull request by cquiroz**
This PR enables phoenix to be usable on the QPT, see below:

screenshot 2016-11-04 13 09 59

While testing an NPE was detected when closing. The second commit adds a fix for that event

### CODE REVIEW

File: bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/variant/VariantDecorator.java [hunk 1]
```diff
@@ -14,72 +14,67 @@
...
-			} else {
-				label.setIcon(ICON_VARIANT); 	
-			}
-			
-			// Alignment
-			label.setHorizontalAlignment(SwingConstants.LEFT);
-			
-		} else if (subElement == VariantAttribute.Wind) {
-
-			if (value == null) value = "-";
```
Comment (swalker2m) [inline bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/variant/VariantDecorator.java]: It seems like in a couple of places if `value` is `null` then we would reassign it to `"-"`.   The logic is a bit hard to follow there in the original code but is this no longer necessary?

Comment (cquiroz) [inline bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/variant/VariantDecorator.java]: I'm not sure why it was there in the first place but the reassignment of `value` was never used
